tests/smoke.runner.js

```diff
@@ -156,6 +147,22 @@ const sleep = (ms) => new Promise(resolve => setTimeout(resolve, ms));
     if (fs.existsSync(logfiles[1])) {
         throw Error(`Expected ${logfiles[1]} to not exist but it does`)
     }
+}
+
+(async () => {
+    const tests = [
+        mochaTestrunner(),
+        jasmineTestrunner(),
+        cucumberTestrunner(),
+        cucumberFailAmbiguousDefinitions(),
+        customService(),
+        customReporterString(),
+        customReporterObject(),
+        multiremote(),
+        retryFail(),
+        retryPass(),
+    ]
+    await Promise.all(tests)
```

Comment:
I rather prefer to run these tests sequentially so in case one test is failing it doesn't make the stdout too noisy

Reply:
in case some of these fail we can easily comment out others and re-run, it runs more than two times faster!

Reply:
are there any other options that can work?

Reply:
> in case some of these fail we can easily comment out others and re-run

Yeah but this doesn't work in CI

> are there any other options that can work?

Maybe allow to run a specific smoke test by applying a cli parameter?

Reply:
> Yeah but this doesn't work in CI

what if we run tests sequentially in CI? Does Travis CI has environment variable so that I can understand if we running test in CI or not?

Reply:
Added condition `if (process.env.CI)` to run sequentially.

Need to make sure that we have that env variable

Reply:
yeah, it works!